content/contributing/core-team/contents.lr

```diff
@@ -0,0 +1,86 @@
+_model: page
+---
+title: Core Team, and Responsibilities
+---
+sort_key: 4
+---
+summary: What is the core team, and what do they do?
+---
+body:
+
+Those busy bees on the `Core Team </community/team>`_ have a number of responsibilities to keep the hive that is BeeWare moving. This is an evolving project, so this page is subject to change.
+
+These include, but aren't limited to, responding to issues, reviewing and merging code, and architecture of the BeeWare project as a whole. 
+
+
+There are people we trust to make code decisions, there are people we trust to make code and organizational decisions, and there is a person with whom the metaphorical buck stops, who guides the vision of the whole organization and is often it's biggest cheerleader
+
+These levels can be described as follows: 
+
+Bee, or Worker Bee:
+ - Any member of the BeeWare community. Given we work in the open on GitHub, anyone can suggest changes to the code and have their code merged. The only limit to your ability to contribute is having your work merged by a team member with the rights to do so. 
+
+Apiarist:
+ - Any Bee with a Commit Bit. These members have demonstrated ability in relation to the project -- this could be on a technical level (JavaScript, Python, Objective-C expertise; GTK, macOS knowledge), or on another level (community management, code review). 
```

Comment:
Point of order - would we still give the commit bit to someone who *doesn't* have the ability to merge code? A really great community organiser, for example, doesn't *need* the commit bit. As currently configured, there isn't an Apiarists team on Github - only Senior Apiarists. Should we have one, but not give that group commit access to anything?

In that vein, would it be better to rephrase "apiarist" as:

A bee who has been recognised as a trusted contributor. These bees have demonstrated ability in relation to a specific part of the BeeWare project over a period of time. This could be on a technical level (JavaScript, Python, Objective-C expertise; GTK+, macOS knowledge), or on another level (community management, code review). Apiarists may also have the commit bit for the project where their expertise is recognized.